Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundMainViewModel.swift

```diff
@@ -65,6 +65,17 @@ final class PlaygroundMainViewModel: ObservableObject {
         }
     }
 
+    @Published var customPublicKey: String = PlaygroundUserDefaults.customPublicKey {
+        didSet {
+            PlaygroundUserDefaults.customPublicKey = customPublicKey
+        }
+    }
+    @Published var customSecretKey: String = PlaygroundUserDefaults.customSecretKey {
+        didSet {
+            PlaygroundUserDefaults.customSecretKey = customSecretKey
```

Comment:
feels weird to pass secret key from client. Do we really need this ability to sent whatever key we'd like?

Reply:
@vardges-stripe 

not sure how there is another way for someone to easily (key word) test any set of keys

[dev app has the same function](https://stripejsdevapp.corp.stripe.com/connections-auth-flow)

Reply:
Yeah, but Dev app is internal. Saving the security keys in keychain feels ikky, but I guess it's only for internal devices.